feat(debugger): Add support for debugging tests (#5208) (#7)
manastech/noir: docs/docs/reference/debugger/debugger_repl.md

### Options

| Option                            | Description                                                                         |
| --------------------------------- | ----------------------------------------------------------------------------------- |
| `-p, --prover-name <PROVER_NAME>` | The name of the toml file which contains the inputs for the prover [default: Prover]|
| `--package <PACKAGE>`             | The name of the package to debug                                                    |
| `--print-acir`                    | Display the ACIR for compiled circuit                                               |
| `--deny-warnings`                 | Treat all warnings as errors                                                        |
| `--silence-warnings`              | Suppress warnings                                                                   |

> | `--test-name` <TEST_NAME>         | The name of the test function to debug - which name contains this string            |

Reviewer: ```suggestion
| `--test-name` <TEST_NAME>         | The name (or substring) of the test function to debug             |
```
